### DISCUSSION TITLE
Custom Image in the Header.

### DISCUSSION THREAD

**Discussion by pouyarezvani** [software-mansion/react-native-screens#904] [Ideas]
![demo](https://user-images.githubusercontent.com/32917154/116804168-81f2ec80-aad1-11eb-82ff-812ad938215b.gif)

Is it possible to Implement an Image such as the gif above inside the header, that is collapsible and draggable down?

**Comment by tboba** [comment]
Hi @pouyarezvani, I believe this is not possible yet - you can only change background of the header as a color.
Thus, I'm moving this discussion to the `Ideas` category. I'll keep in touch if anything changes in this area.
